### DISCUSSION TITLE
是否可能使代理出站将域名解析为IP？希望类似"domainStrategy": "UseIP"的选项。

### DISCUSSION THREAD

**Discussion by RealXuChe** [v2fly/v2ray-core#2948] [Q&A] [answered]
就我理解，目前只有在Freedom出站上，v2ray才会解析域名**用于连接**。在如`Shadowsocks`等代理出站上，v2ray将解析的IP用于路由，但是向代理服务器发出的请求中，**目标地址还是域名**。

因为我使用的境外代理服务器的DNS似乎屏蔽了一些域名，我认为使用IP请求应该能绕过屏蔽。我希望本地运行的v2ray将收到的代理请求的目标域名解析为IP，向境外代理服务器发出IP代理请求，我该怎么做？

**Comment by dyhkwong** [comment, marked as answer]
#2553 该功能已在 v5.12.0 被支持。

**Reply by RealXuChe** [reply]
谢谢！但是似乎这个功能并没有记载在v2fly.org上的文档中。这个功能是没有被详细记载，还是说别处有更详尽的资料？

**Comment by dyhkwong** [comment]
文档缺乏维护，建议以源码为准
